libs/model/src/contest/CreateContestManagerMetadata.command.ts

```diff
@@ -44,6 +44,8 @@ export function CreateContestManagerMetadata(): Command<
         }));
       }
 
+      const farcaster_frame_url = `/api/farcaster/${payload.contest_address}`;
```

Comment:
would this lead to our api? like `localhost:3000/api/farcaster/0x123`? 
asking because we have already router hooked up to `api/farcaster` [endpoint](https://github.com/hicommonwealth/commonwealth/blob/4d668b485f6b8dbdbb1b5f04d8505202782c530f/packages/commonwealth/server/routing/router.ts#L1128) that handles farcaster frames

Reply:
Yes, that URL will point to the farcaster frame endpoint and be rendered to the frontend so that the user can copy/paste it. We're using your existing endpoint for the target.